src/__tests__/main-test.ts

```diff
@@ -4,48 +4,17 @@ import listen = require("test-listen");
 import { URL } from "url";
 import handler = require("../main");
 
-describe("GET /", () => {
-  test("responds with value that is given at `message` query parameter", async () => {
+describe("GET /graphql", () => {
+  test("responds graphql data", async () => {
```

Comment:
I think this is much better

```suggestion
  test("responds data through graphql", async () => {
```